deploy/ansible/roles/nginx/templates/site.conf

```diff
@@ -1,5 +1,6 @@
 upstream app {
   server 0.0.0.0:8000;
+  root {{public_path}};
```

Comment:
have you tried this? i don't *think* that will work. we want this down in the server block